docs/deployment_patterns/how_to_use_gx_with_aws/components/_expectation_suite_batch_request_validator_prepare_or_reuse.md

```diff
@@ -0,0 +1,3 @@
+When we tested our Datasource in step [2.5: Test your new Datasource](how_to_use_gx_with_aws_using_cloud_storage_and_pandas#25-test-your-new-datasource) we also created all of the components we need to begin creating Expectations: A Batch Request to provide sample data we can test our new Expectations against, an empty Expectation Suite to contain our new Expectations, and a Validator to create those Expectations with.
+
```

Comment:
For the `RuntimeDataConnector` and `RuntimeBatchRequest`, I think it might be worth mentioning that we want to `S3 URI` Rather than ARN or Object URL etc.